docs/current/guides/324301-argo-workflows.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: /324301/argo-workflows
+displayed_sidebar: "current"
+category: "guides"
+tags: ["argo", "kubernetes"]
+authors: ["Kyle Penfound"]
+date: "2023-08-22"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Use Dagger with Argo Workflows
+
+## Introduction
+
+[Argo Workflows](https://argoproj.github.io/argo-workflows/) is an open source container-native workflow engine for orchestrating parallel jobs on Kubernetes. This guide explains how to run Dagger pipelines in Argo Workflows.
+
+## Requirements
+
+This guide assumes that you have a basic understanding of Kubernetes and Argo Workflows, and that your Kubernetes cluster has been configured following the [Run Dagger on Kubernetes](https://docs.dagger.io/194031/kubernetes) guide.
+
+## Step 1: Install Argo Workflows
+
+The first step is to install [Argo Workflows](https://argoproj.github.io/argo-workflows/) in the Kubernetes cluster.
+
+Follow the [Argo Workflows quickstart](https://github.com/argoproj/argo-workflows/blob/master/docs/quick-start.md) steps, adjusting them as needed to your own requirements. Once you've successfully installed Argo Workflows in your cluster, continue to the next step.
```

Comment:
Since we are using the `argo` CLI further below, we should explicitly require this to be installed too.

Reply:
If you think it's necessary, sure. In the guide we instruct them to follow it has a step for installing the `argo` CLI https://github.com/argoproj/argo-workflows/blob/master/docs/quick-start.md#install-the-argo-workflows-cli

Reply:
I've spotted it now, all good!